dist/index.js

```diff
@@ -49,9 +50,20 @@ class Resolver {
      * @returns {Promise<AccountId>}
      */
     async resolveSLD(domain) {
+        const udResolution = new resolution_1.Resolution();
         const nameHash = (0, hashDomain_1.hashDomain)(domain);
         const sld = await this.getSecondLevelDomain(nameHash);
```

Comment:
Isn't `getSecondLevelDomain` going to throw for an Unstoppable Domain like `aaronquirk.x`, since it won't be found on the topic? It seems like the error needs to be handled and then UD `isSupportedDomain` checked if a Hashgraph.name domain is not found.

Reply:
This is a good poing @qrtp, I'm happy to fix this though, whats most helpful is that your side is writing the resolution for UD domains.